attributes/default.rb

```diff
@@ -26,21 +26,36 @@
-default['rundeck_server']['rdeck_jvm']['rdeck.config']                    = node['rundeck_server']['confdir']
-default['rundeck_server']['rdeck_jvm']['rundeck.server.configDir']        = node['rundeck_server']['confdir']
-default['rundeck_server']['rdeck_jvm']['rundeck.server.serverDir']        = node['rundeck_server']['basedir']
-default['rundeck_server']['rdeck_jvm']['rdeck.base']                      = node['rundeck_server']['basedir']        # Default is the directory containing the launcher jar
-default['rundeck_server']['rdeck_jvm']['server.http.host']                = '0.0.0.0'                                    # Address/hostname to listen on, default is "0.0.0.0"
-default['rundeck_server']['rdeck_jvm']['server.http.port']                = node['rundeck_server']['port']['http']   # The HTTP port to use for the server, default "4440"
-default['rundeck_server']['rdeck_jvm']['server.https.port']               = node['rundeck_server']['port']['https']  # The HTTPS port to use or the server, default "4443"
-default['rundeck_server']['rdeck_jvm']['java.io.tmpdir']                  = ::File.join('tmp', 'rundeck')
-default['rundeck_server']['rdeck_jvm']['server.datastore.path']           = ::File.join(node['rundeck_server']['basedir'], 'data')                  # Path to server datastore dir
-default['rundeck_server']['rdeck_jvm']['java.security.auth.login.config'] = ::File.join(node['rundeck_server']['confdir'], 'jaas-loginmodule.conf')
-default['rundeck_server']['rdeck_jvm']['rdeck.projects']                  = ::File.join(node['rundeck_server']['datadir'], 'projects')
-default['rundeck_server']['rdeck_jvm']['rdeck.runlogs']                   = ::File.join(node['rundeck_server']['basedir'], 'logs')
-default['rundeck_server']['rdeck_jvm']['rundeck.config.location']         = ::File.join(node['rundeck_server']['confdir'], 'rundeck-config.properties')
+# JVM configuration
+#
+# Option mapping rules
+# ['key'] = 'string' maps to -key=string
+# ['key'] = boolean  maps to -key if boolean is true
+#
+# System properties (D prefix)
+default['rundeck_server']['jvm']['Dloginmodule.name']                = 'RDpropertyfilelogin'
+default['rundeck_server']['jvm']['Drdeck.config']                    = node['rundeck_server']['confdir']
+default['rundeck_server']['jvm']['Drundeck.server.configDir']        = node['rundeck_server']['confdir']
+default['rundeck_server']['jvm']['Drundeck.server.serverDir']        = node['rundeck_server']['basedir']
+default['rundeck_server']['jvm']['Drdeck.base']                      = node['rundeck_server']['basedir']       # Default is the directory containing the launcher jar
+default['rundeck_server']['jvm']['Dserver.http.host']                = '0.0.0.0'                               # Address/hostname to listen on, default is "0.0.0.0"
+default['rundeck_server']['jvm']['Dserver.http.port']                = node['rundeck_server']['port']['http']  # The HTTP port to use for the server, default "4440"
```

Comment:
Yes, because the init script has the following line:
rundeckd="${JAVA_HOME:-/usr}/bin/java ${RDECK_JVM} -cp ${BOOTSTRAP_CP} com.dtolabs.rundeck.RunServer /var/lib/rundeck ${RDECK_HTTP_PORT}"